analysis/wwz/make_sample_lst_json.py

```diff
@@ -151,14 +151,16 @@
 }
 central_2022_dict = {
     "2022_tWZ4l" : {
-        "path" : "/store/user/t2/users/matthew.dittrich/skims/Run3_MC_4L_04022024_Loose/TWZ_TtoLNu_WtoLNu_Zto2L_DR2_TuneCP5_13p6TeV_amcatnlo-pythia8_Run3Summer22NanoAODv12-130X_mcRun3_2022_realistic_v5-v3_NANOAODSIM_Run3_MC_4L_04022024_Loose/",
+        #"path" : "/store/user/t2/users/matthew.dittrich/skims/Run3_MC_4L_04022024_Loose/TWZ_TtoLNu_WtoLNu_Zto2L_DR2_TuneCP5_13p6TeV_amcatnlo-pythia8_Run3Summer22NanoAODv12-130X_mcRun3_2022_realistic_v5-v3_NANOAODSIM_Run3_MC_4L_04022024_Loose/",
```

Comment:
Why comment this out? Can you add a comment explaining it, or just remove the unused one?

Reply:
(I don't think any change to the code or even comment in the code is needed, but a comment here on the PR would be appreciated thanks!)

Reply:
I think we should go ahead and keep the DR2 sample path. There were some initial questions about which one we should use. I have added a comment about this